spacy/lemmatizer.py

```diff
@@ -86,13 +86,16 @@ def lemmatize(string, index, exceptions, rules):
     #if string in index:
     #    forms.append(string)
     forms.extend(exceptions.get(string, []))
+    oov_forms = []
```

Comment:
Maybe this needs to be:

    oov_forms = [string]

The tests are currently failing when we don't match any rules.